### PR TITLE
Added twerion

### PR DESCRIPTION
**Pull request by totKing**
### General:
* [X] The pull request title is descriptive. *(ex. `Added GoldenPvP Network`, not `Updated metadata.json`)*
* [X] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [X] Folder with appropriate name has been created / updated (must not include any non-alpha characters or whitespace).
* [x] No changes were made to the file's formatting (4 spaces per tab).
* [x] There are no syntax errors.
* [X] There is no pre-existing mapping matching my id (check if there is an existing folder).
* [X] My field values match your requirements.
* You can view our patterns here: [metadata.schema.json](https://github.com/CheatBreakerNet/Client-API/blob/master/mappings/metadata.schema.json), or take a look below and complete the field checklist:
  - [X] `id`: a lowercase string, which should match the folder name *(ex. `goldenpvpnetwork`)*
  - [X] `name`: a string *(ex. `GoldenPvP Network`)*
  - [X] `description`: hook between 16 and 40 characters *(ex. `We're the home of classic PvP gamemodes`)* 
  - [X] `addresses`: an array with lowercase strings *(ex. `["goldenpvp.net", "routing.center"]`)*
    - You do not need to specify subdomains, CheatBreaker services automatically detect them.
  - [X] `primaryAddress`: the primary address that people connect to with (please include the subdomain if required) *(ex. `mc.goldenpvp.net`)*
  - [X] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.7.*", "1.8.9"]` - Must be versions or subversions supported by CheatBreaker)*
  - [X] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.7.10` - Must be a subversion supported by CheatBreaker)*
  - [X] `primaryColor`: a hexademical color code that primarily distinguishes the server *(ex. `#00FFFF`)* 
  - [X] `secondaryColor`: a hexademical color code that accompanies the `primaryColor` of the server *(ex. `#FF0000`)*
  - [X] `primaryRegion`: the primary region where your server operates in *(ex. `EU`)*
  - [X] `regions`: a list of regions where you have servers located that service your players *(ex. `["EU", "NA", "AS"]`)*
  - [X] `gameTypes`: a list of games that describe the content on your server, must be a max of 3 listed *(ez. `["PVP", "UHC", "HCF"]`)*
  - [X] (optional) `website`: url of server website, must include URL schema (http:// or https://) *(ex. `https://www.goldenpvp.net`)*
  - [X] (optional) `store`: url of server store, must include URL schema (http:// or https://) *(ex. `https://store.goldenpvp.net`)*

### Socials
* [X] (optional) `twitter`: username of twitter account without the @ *(ex. GoldenPvPNet)*
* [X] (optional) `discord`: invite link to discord *(ex. https://discord.com/invite/3EAzBxf)*
* [X] (optional) `youtube`: slug or username of youtube channel *(ex. GoldenPvP)*
* [X] (optional) `instagram`: username of instagram account *(ex. GoldenPvPNet)*
* [ ] (optional) `twitch`: username of twitch account *(ex. GoldenPvPNet)*
* [ ] (optional) `telegram`: slug of telegram group *(ex. GoldenPvP-Network)*
* [ ] (optional) `reddit`: slug of subdreddit wtih 'r/' *(ex. GoldenPvP)*
* [X] (optional) `tiktok`: username of tiktok account *(ex. GoldenPvP)*
* [ ] (optional) `facebook`: slug of facebook page *(ex. GoldenPvP)*

#### Icon
* [X] My icon is a `png` file.
* [X] I have uploaded my icon to my server folder *(ex. `goldenpvpnetwork`)* and named it `icon.png`.
* [X] My icon has a transparent background and is square (1:1 aspect ratio).
* [X] My icon is `64` pixels in width and height.
* [X] My icon is my own/the server's property and complies with relevant copyright/privacy laws.

#### Logo
* [X] My logo is a `png` file.
* [X] I have uploaded my logo to my server folder *(ex. `goldenpvpnetwork`)* and named it `logo.png`.
* [X] My logo has a transparent background and is square (1:1 aspect ratio).
* [X] My logo is `108` pixels in width and height.
* [X] My logo is my own/the server's property and complies with relevant copyright/privacy laws.

#### Discord Logo
* [X] My logo is a `png` file.
* [X] I have uploaded my logo to my server folder *(ex. `goldenpvpnetwork`)* and named it `discord.png`.
* [X] My logo has a transparent background and is square (1:1 aspect ratio).
* [X] My logo is between `512` and `1024` pixels in width and height.
* [X] My logo is my own/the server's property and complies with relevant copyright/privacy laws.

#### Primary Background (Only required for partnered servers)
* [ ] My primary background is a `png` file.
* [ ] I have uploaded my primary background to my server folder *(ex. `goldenpvpnetwork`)* and named it `primarybackground.png`.
* [ ] My primary background is `774` pixels in width and `363` pixels in height.
* [ ] My primary background contains relevant content pertaining to the server, is my own/the server's property, and complies with relevant copyright/privacy laws.
* [ ] My primary background doesn't contain any markings, text, or logos (unless part of a build).

#### Secondary Background (Only required for partnered servers)
* [ ] My secondary background is a `png` file.
* [ ] I have uploaded my secondary background to my server folder *(ex. `goldenpvpnetwork`)* and named it `secondarybackground.png`.
* [ ] My secondary background is `447` pixels in width and `172` pixels in height.
* [ ] My secondary background contains relevant content pertaining to the server, is my own/the server's property, and complies with relevant copyright/privacy laws.
* [ ] My secondary background doesn't contain any markings, text, or logos (unless part of a build).

### CODE REVIEW

**Review by Tellinq — CHANGES_REQUESTED** [review]
This version needs to be one we already support

File: mappings/servers/twerion/metadata.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      "1.11.*",
+      "1.12.*",
+      "1.16.*",
+      "1.17.*",
+      "1.18.*",
+      "1.19.*",
+      "1.20.*",
+      "1.21.*"
+   ],
+   "primaryMinecraftVersion":"1.8.8",
```
Comment (Tellinq) [inline mappings/servers/twerion/metadata.json:26]: The primary version needs to be one we already support

**Review by Tellinq — CHANGES_REQUESTED** [review]

File: mappings/servers/twerion/metadata.json (hunk 1 above)
Comment (totKing) [reply mappings/servers/twerion/metadata.json]: Has been changed.

File: mappings/servers/twerion/metadata.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    "name": "Twerion Network",
+    "description": "A network focused on PvP",
+    "addresses": [
+        "twerion.net",
+        "twerion.de",
+        "twerion.com",
+        "twerion.eu",
+        "twerion.org"
+    ],
+    "primaryAddress": "cb.twerion.net",
```
Comment (Tellinq) [inline mappings/servers/twerion/metadata.json:12]: What’s the purpose to have a CB specific primary address? We typically accept primary addresses for the whole server, not CB specific primary addresses.

Comment (Hyronymos1337) [reply mappings/servers/twerion/metadata.json:12]: Join tracking, so we know exactly how many people are joining from the Cheatbreaker launcher.

Comment (totKing) [reply mappings/servers/twerion/metadata.json:12]: We can also change it if desired.

Comment (Tellinq) [reply mappings/servers/twerion/metadata.json:12]: The proper way to track CheatBreaker users is through the https://github.com/CheatBreakerNet/Bukkit-API. We have a dedicated plugin message you can use to check.

Besides, end users on other clients can just use this address to skew your intended tracking system

I'd advise just using the normal address instead if the justification is just this improper tracking system

Comment (Hyronymos1337) [reply mappings/servers/twerion/metadata.json]: Thanks for the quick reply, we will do it via CheatBreaker API.

Comment (totKing) [reply mappings/servers/twerion/metadata.json]: We have changed it back to the normal domain.

**Review by Tellinq — APPROVED** [review]

**Review by Tellinq — APPROVED** [review]
Reapproving. Just need to make sure Actions successfully runs

File: mappings/servers/twerion/metadata.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        "nl",
+        "pl",
+        "pt",
+        "ro",
+        "ru",
+        "se",
+        "sk",
+        "sl",
+        "tr"
+    ],
```
Comment (Tellinq) [inline mappings/servers/twerion/metadata.json:46]: I completely missed this. The format for these languages do not match our required scheme. If you feel like we're missing some languages, let me know and I can consider getting them added to our scheme
image

**Review by Tellinq — CHANGES_REQUESTED** [review]

**Review by Tellinq — CHANGES_REQUESTED** [review]
Now the entirety of the metadata file doesn’t follow the spacing guidelines (4 spaces is required, not 3)

**Comment by totKing** [comment]
> Now the entirety of the metadata file doesn’t follow the spacing guidelines (4 spaces is required, not 3)

I have corrected it.

File: mappings/servers/twerion/metadata.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        "el",
+        "ja",
+        "ko",
+        "lt",
+        "no",
+        "pl",
+        "pt-PT",
+        "pt-BR",
+        "sv-SE"
+    ],
```
Comment (Tellinq) [inline mappings/servers/twerion/metadata.json:39]: Something tells me you don't actually support these languages and might be listing for the sake of listing. Please ensure you're keeping this list accurate. We only accept languages that are truly supported.

image

**Review by Tellinq — CHANGES_REQUESTED** [review]

File: mappings/servers/twerion/metadata.json (hunk 4 above)
Comment (Hyronymos1337) [reply mappings/servers/twerion/metadata.json:39]: Why should we add 30 website translations when every browser has an auto translator built in?

Just do /language on the twerion.net server.

image

Comment (Tellinq) [reply mappings/servers/twerion/metadata.json:39]: Alright, if that’s there, I’ll accept it.

**Review by Tellinq — APPROVED** [review]

File: mappings/servers/twerion/metadata.json [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        "AS",
+        "NA",
+        "OC",
+        "SA"
+    ],
+    "gameTypes": [
+        "PvP",
+        "Minigames",
+        "Skyblock"
+    ],
```
Comment (Tellinq) [inline mappings/servers/twerion/metadata.json:53]: Our scheme requires `SkyBlock`, not `Skyblock`

**Review by Tellinq — CHANGES_REQUESTED** [review]

**Review by Tellinq — APPROVED** [review]
Approving for Actions

**Pull request merged**